appveyor.yml

```diff
@@ -48,6 +48,7 @@ install:
   - echo extension=php_pgsql.dll >> php.ini
   - echo extension=php_gd2.dll >> php.ini
   - echo extension=php_fileinfo.dll >> php.ini
+  - echo memory_limit=256M >> php.ini
```

Comment:
Should we just use -1 here?

Reply:
Not sure. The default was 128M. `PharData` ate all memory on compression